sources/packages/web/src/views/institution/InstitutionDashboard.vue

```diff
@@ -1,9 +1,12 @@
 <template>
-  <h1 class="d-flex justify-content-center">Dashboard Work in Progress</h1>
+  <formio :formName="'institutionDashboard'"></formio>
```

Comment:
the path in the formio is institutiondashboard, please change it with the proper case

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/62901416/134596863-ca17edf3-9b59-4115-aa0d-80b6d7de13ab.png">